notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<p>Does this make it easier to see data? Or are you just trying to create a "publication-ready" plot? Either way, you should explain why you want to do this</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>